samples/wifi/radio_test/src/nrf_wifi_radio_ficr_shell.c

```diff
@@ -0,0 +1,345 @@
+	else
+		LOG_INF("CALIB_MAXPOW5G0MCS7 is programmed in OTP\n");
+
+	if (flags & (~CALIB_MAXPOW5G0MCS0_FLAG_MASK))
+		LOG_INF("CALIB_MAXPOW5G0MCS0 is not programmed in OTP\n");
+	else
+		LOG_INF("CALIB_MAXPOW5G0MCS0 is programmed in OTP\n");
+
+	if (flags & (~CALIB_RXGAINOFFSET_FLAG_MASK))
+		LOG_INF("CALIB_RXGAINOFFSET is not programmed in OTP\n");
+	else
+		LOG_INF("CALIB_RXGAINOFFSET is programmed in OTP\n");
+
+	if (flags & (~CALIB_TXPOWBACKOFFT_FLAG_MASK))
+		LOG_INF("CALIB_TXPOWBACKOFFT is not programmed in OTP\n");
+	else
+		LOG_INF("CALIB_TXPOWBACKOFFT is programmed in OTP\n");
+
+	if (flags & (~CALIB_TXPOWBACKOFFV_FLAG_MASK))
+		LOG_INF("CALIB_TXPOWBACKOFFV is not programmed in OTP\n");
+	else
+		LOG_INF("CALIB_TXPOWBACKOFFV is programmed in OTP\n");
+}
+
+
+static int nrf_wifi_radio_test_otp_get_status(const struct shell *shell,
+					size_t argc,
+					char  *argv[])
+{
+	unsigned int ret, err;
+	unsigned int val[OTP_MAX_WORD_LEN];
+
+	/* read all the OTP memory */
+	err = read_otp_memory(0, &val[0], OTP_MAX_WORD_LEN);
+	if (err) {
+		LOG_ERR("FAILED reading otp memory......\n");
+		return -ENOEXEC;
+	}
+
+	LOG_INF("Checking OTP PROTECT Region......\n");
+	ret = check_protection(&val[0], REGION_PROTECT, REGION_PROTECT+1,
+							REGION_PROTECT+2, REGION_PROTECT+3);
+
+	disp_location_status("OTP", ret);
+	disp_fields_status(val[REGION_DEFAULTS]);
+
+	return 0;
+}
+
+static int nrf_wifi_radio_test_otp_read_params(const struct shell *shell,
+					size_t argc,
+					char  *argv[])
+{
+	unsigned int val[OTP_MAX_WORD_LEN];
+	unsigned int ret, err;
+
+
+	err = read_otp_memory(0, &val[0], OTP_MAX_WORD_LEN);
+	if (err) {
+		LOG_ERR("FAILED reading otp memory......\n");
+		return -ENOEXEC;
+	}
+
+	ret = check_protection(&val[0], REGION_PROTECT+0, REGION_PROTECT+1,
+							REGION_PROTECT+2, REGION_PROTECT+3);
+
+	disp_location_status("OTP", ret);
+
+	LOG_INF("REGION_PROTECT0 = 0x%08x\n", val[REGION_PROTECT+0]);
+	LOG_INF("REGION_PROTECT1 = 0x%08x\n", val[REGION_PROTECT+1]);
+	LOG_INF("REGION_PROTECT2 = 0x%08x\n", val[REGION_PROTECT+2]);
+	LOG_INF("REGION_PROTECT3 = 0x%08x\n", val[REGION_PROTECT+3]);
+	LOG_INF("\n");
+
+	LOG_INF("MAC0: Reg0 = 0x%08x\n", val[MAC0_ADDR]);
+	LOG_INF("MAC0: Reg1 = 0x%08x\n", val[MAC0_ADDR+1]);
+	LOG_INF("MAC0 Addr  = %02x:%02x:%02x:%02x:%02x:%02x\n",
+		(uint8_t)(val[MAC0_ADDR]), (uint8_t)(val[MAC0_ADDR]>>8),
+		(uint8_t)(val[MAC0_ADDR]>>16), (uint8_t)(val[MAC0_ADDR]>>24),
+		(uint8_t)(val[MAC0_ADDR+1]), (uint8_t)(val[MAC0_ADDR+1]>>8));
+	LOG_INF("\n");
+
+	LOG_INF("MAC1 : Reg0 = 0x%08x\n", val[MAC1_ADDR]);
+	LOG_INF("MAC1 : Reg1 = 0x%08x\n", val[MAC1_ADDR+1]);
+	LOG_INF("MAC1 Addr   = %02x:%02x:%02x:%02x:%02x:%02x\n",
+		(uint8_t)(val[MAC1_ADDR]), (uint8_t)(val[MAC1_ADDR]>>8),
+		(uint8_t)(val[MAC1_ADDR]>>16), (uint8_t)(val[MAC1_ADDR]>>24),
+		(uint8_t)(val[MAC1_ADDR+1]), (uint8_t)(val[MAC1_ADDR+1]>>8));
+	LOG_INF("\n");
+
+	LOG_INF("CALIB_XO = 0x%02x\n", val[CALIB_XO]&0xff);
+	LOG_INF("\n");
```

Comment:
Again, are those empty lines in between really needed?

Reply:
Yes, readability